src/app/@core/styles/themes.scss

```diff
@@ -115,9 +117,12 @@ $nb-themes: nb-register-theme((font-family-primary: unquote("Comfortaa, sans-ser
         text-heading-4-font-size: 1.125rem,
         text-heading-5-font-size: 1.125rem,
         text-heading-6-font-size: 1.125rem,
-        search-background-color: 'rgba(255, 255, 255, 0.9)'
+        search-background-color: "rgba(255, 255, 255, 0.9)",
```

Comment:
couldn't we use/create one more `color-basic-control-transparent-x` for this purpose?

Reply:
I did create a new one, color-basic-transparent-900